misc/gene-update/backfill_gene_id.py

```diff
@@ -7,6 +7,7 @@
 
 import uta
 from uta.models import Gene, Transcript
+from uta.tools.file_utils import open_file
```

Comment:
Nice, this will be useful in a lot of other places

Reply:
Thanks to @sptaylor!